src/scripting/zscript/zcc_parser.cpp

```diff
@@ -415,6 +415,24 @@ static void DoParse(int lumpnum)
 	for (unsigned i = 0; i < Includes.Size(); i++)
 	{
 		lumpnum = Wads.CheckNumForFullName(Includes[i], true);
+		if (lumpnum == -1 && Includes[i].IndexOf("./") == 0)//relative path resolving
+		{
+			FString fullPath = IncludeLocs[i].FileName.GetChars();//get full path, format 'wad:filepath/filename'
+			
+			size_t start = fullPath.IndexOf(":");//find first ':'
+
+			size_t end = fullPath.LastIndexOf("/");//find last '/'
+
+			if(start!=-1&&end!=-1)
```

Comment:
Could you please use spaces? `if (start != -1 && end != -1)` is much easier to read.